.github/actions/build-schemas/action.yaml

```diff
@@ -0,0 +1,33 @@
+name: Get schemas
+description: Gets new schemas from cluster
+runs:
+  using: composite
+  steps:
+    - name: Authenticate and set context
+      uses: redhat-actions/oc-login@v1
+      env:
+        OPENSHIFT_NAMESPACE: default
+      with:
+        openshift_server_url: ${{ inputs.OPENSHIFT_SERVER }}
+        openshift_token: ${{ inputs.OPENSHIFT_TOKEN }}
+        insecure_skip_tls_verify: true
+    - name: Update schemas
+      shell: bash
+      run: |
+        pip install openapi2jsonschema
+        cd $GITHUB_WORKSPACE
+        python build_schema.py -u $(oc whoami --show-server) -t $(oc whoami -t) -s STRICT
+        oc logout
+    - name: Upload artifacts from generating schemas
+      uses: actions/upload-artifact@v3
+      with:
+        name: schemas
+        path: |
+          schemas/
+        retention-days: 1
+    - name: Upload artifacts from generating schemas
+      uses: actions/upload-artifact@v3
+      with:
+        path: |
+          skip_kinds.txt
+        retention-days: 1
```

Comment:
Can't we join these two steps into one?

https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions

Reply:
This was done to make it easier to extract `skip_kinds.txt` from artifacts folder since in the repository it is located in the root, but when I am thinking about it it should be also possible to extract it from the schemas artifact folder, will try it out

Reply:
Joined uploads together, had to also add script to move `skip_kinds.txt` and other files out of artifacts folder because this lead to nested structure which was not wanted in the PR